### PR TITLE
v0.15 prepatations

### PR DESCRIPTION
**Pull request by meyskens**
Preparations for the v0.15 release including release notes and upgrade notes.

### CODE REVIEW

File: content/en/docs/release-notes/release-notes-0.15.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        name: pkcs12-password-secret
+```
+
+For JKS this adds `keystore.jks` and `truststore.jks` and `keystore.p12` for PKCS#12.
+
+## cert-manager-ctl CLI tool
+
+cert-manager-ctl is a CLI tool that assists with controlling cert-manager inside your
+Kubernetes cluster. The cert-manager-ctl binary can be downloaded from the [GitHub release page](https://github.com/jetstack/cert-manager/releases/tag/v0.15).
+In v0.15 the use is currently limited to the `convert` and `renew` command.
```
Comment (JamesLaverack) [inline content/en/docs/release-notes/release-notes-0.15.md:112]:
```suggestion
In v0.15 the use is currently limited to the `convert` and `renew` commands.
```

File: content/en/docs/release-notes/release-notes-0.15.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+type: "docs"
+---
+
+The `v0.15` release has a few focus areas:
+
+* Experimental improved controllers
+* Support for OpenShift's OLM
+* Improved deployment of the webhook
+* General Availability of JKS and PKCS#12 keystores
+* cmctl CLI tool
```
Comment (munnerz) [inline content/en/docs/release-notes/release-notes-0.15.md:14]: Need to remove references to the name `cmctl`

File: .spelling [hunk 3]
```diff
@@ -144,3 +144,11 @@ v3
 publicised
 analyse
 PKCS#12
+OperatorHub
+keystores
+keystores
+keystore
+config
+cmctl
```
Comment (munnerz) [inline .spelling:152]: Not a valid word anymore :)

File: .spelling [hunk 4]
```diff
@@ -144,3 +144,11 @@ v3
...
+RedHat
```
Comment (munnerz) [inline .spelling:153]: Can we just refer to `Red Hat` instead?

Comment (meyskens) [reply .spelling:153]: I always was so convinced it was RedHat.... turns out you should never trust yourself

File: content/en/docs/release-notes/release-notes-0.15.md (hunk 2 above)
Comment (meyskens) [reply content/en/docs/release-notes/release-notes-0.15.md]: Renamed

File: .spelling (hunk 3 above)
Comment (meyskens) [reply .spelling]: removed

**Pull request merged**